src/options/README.md

```diff
@@ -59,6 +59,16 @@ Optional attributes are:
   given)
 
 
+Option categories
+-----------------
+
+Every option has one of the following categories that influences where and how an option is visible:
+
+* `common`: used for the most common options. All `common` options are shown at the very top in both the online documentation and the output of `--help` on the command line.
+* `regular`: this should be used for most options.
+* `expert`: used for options that should be used with care only. A warning is shown in both the online documentation and the command line help.
```

Comment:
```suggestion
* `expert`: This is for options that should be used with care only. A warning is shown in both the online documentation and the command line help.
```

Reply:
Maybe clarify what kind of warning? Without context this might be a bit weird for users.